make-go.bash

```diff
@@ -60,6 +60,11 @@ mkdir -p "$GOROOT_FINAL"
 
 pushd ext/golang/go/src
 
+for PATCH in ../../../../patches/golang/all/*.patch; do
+    echo "Applying $PATCH"
+    patch -p1 <$PATCH
```

Comment:
Running `./gradlew cleanNative buildNative` gave me [this error](https://gist.github.com/Nutomic/b92f0b3c29fa87c619bdfa63ba5c4f1d), because `make-go.bash` is called twice (for each arch). You'll have to use `patch ... || true` to ignore additional calls (and please add a short comment explaining this).